analysed-packages/kmod/version-v29/kmod-v29-SPDX2TV.spdx

```diff
@@ -0,0 +1,7184 @@
+FileChecksum: SHA256: efa41921e11f60110b3f7d74f1e309a2f95de6ba37041f7f38fe5d6b1067e066
+FileChecksum: MD5: 2d2ec8f3b485e8a22c2f1dbf639c4f1e
+LicenseConcluded: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-gpl-2.0-plus
+FileCopyrightText: <text> Copyright (C) 2015 Intel Corporation. All rights reserved.
+Copyright (C) 2011-2013 ProFUSION embedded systems </text>
+
+##File
+
+FileName: kmod-29.tar.gz/kmod-29.tar/kmod-29/tools/modinfo.c
+SPDXID: SPDXRef-item522318
+FileChecksum: SHA1: 301d50bdce47a53cf11199ce6d88a9b2bf7b46cb
+FileChecksum: SHA256: 93dbd3bb149649717ad8740873533fdbedbd2b6b6e0b9d6a8e577f74d0bd4f1c
+FileChecksum: MD5: 61c3d449acf7f258ca729ae48f2d786f
+LicenseConcluded: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-gpl-2.0-plus
+FileCopyrightText: <text> Copyright (C) 2011-2013 ProFUSION embedded systems </text>
+
+##File
+
+FileName: kmod-29.tar.gz/kmod-29.tar/kmod-29/tools/rmmod.c
+SPDXID: SPDXRef-item522321
+FileChecksum: SHA1: e7a6e211573a7b275cb3656a66693568912db01d
+FileChecksum: SHA256: 1cfdde39bdbf85742750d8af2c850ef6812a928bb6c619f1ec5e61163b05ba91
+FileChecksum: MD5: 44d0692b59d7a6cb1083c08b55c77b75
+LicenseConcluded: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-gpl-2.0-plus
+FileCopyrightText: <text> Copyright (C) 2011-2013 ProFUSION embedded systems </text>
+
+##File
+
+FileName: kmod-29.tar.gz/kmod-29.tar/kmod-29/tools/insmod.c
+SPDXID: SPDXRef-item522315
+FileChecksum: SHA1: 2631771b340b1f266b8a846cd74189295a74322c
+FileChecksum: SHA256: 68c526b5ac81070abe8b532b39d7f0f0029fff91380df0745a6ab7f62827c7f5
+FileChecksum: MD5: b8f35126e4749992000ed6bcd0ca5c92
+LicenseConcluded: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-gpl-2.0-plus
+FileCopyrightText: <text> Copyright (C) 2011-2013 ProFUSION embedded systems </text>
+
+##File
+
+FileName: kmod-29.tar.gz/kmod-29.tar/kmod-29/COPYING
+SPDXID: SPDXRef-item522352
+FileChecksum: SHA1: 545f380fb332eb41236596500913ff8d582e3ead
+FileChecksum: SHA256: 6095e9ffa777dd22839f7801aa845b31c9ed07f3d6bf8a26dc5d2dec8ccc0ef3
+FileChecksum: MD5: a6f89e2100d9b6cdffcea4f398e37343
+LicenseConcluded: LicenseRef-License-of-GNU-Licenses
+LicenseComments: <text> The file contains the text of the LGPL-2.1-or-later. The text of the license is licensed under:
+
+Everyone is permitted to copy and distribute verbatim copies
+of this license document, but changing it is not allowed.
+
+Thus we concluded License-of-GNU-Licenses as license of this file. </text>
+LicenseInfoInFile: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-LGPL-2.1-only
+LicenseInfoInFile: LicenseRef-License-of-GNU-Licenses
+LicenseInfoInFile: LicenseRef-LGPL-2.1
+FileCopyrightText: <text> Copyright (C) 1991, 1999 Free Software Foundation, Inc.
+written by James Random Hacker. </text>
```

Comment:
Please remove this, this is not a real copyright statement. For further details please see my comment in the disclosure document